src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExporter.cs

```diff
@@ -31,7 +31,7 @@ namespace Dynatrace.OpenTelemetry.Exporter.Metrics
 	/// https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/metric-ingestion-protocol/
 	/// https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/post-ingest-metrics
 	/// </summary>
-	[AggregationTemporality(AggregationTemporality.Delta)]
```

Comment:
I feel like there is something missing in the .NET SDK here, or maybe I just wasn't able to find it. I am pretty sure there is a requirement to provide per-instrument configuration of the temporality. Not sure if this is something we can specify in the exporter (it works in Java) but we should probably include it in the Example, if it is even possible

Reply:
As far as I can tell there is now way to let the exporter specify an `AggregationTemporalityPreference` in the .NET SDK. It is probably present in Java as the Specification permits this automatic configuration, but it does not require it. It currently reads:

> OpenTelemetry language implementations MAY support automatically configuring the [MetricReader](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricreader) to use for an Exporter.

Reply:
I think what Georg is saying is that in Java, you can pass a `(instrType) => ` when configuring the reader to say which temporality you want for each instrument type. 

Marc pointed out, they _do_ have such func mapping, but it's internal and it follows the table here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md

By settings to delta, it will use this function, so we should be good:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/33125ac3bc3c5aa96a6a203adf844ebfe912a05b/src/OpenTelemetry/Metrics/MetricReader.cs#L36-L58

Reply:
Oh, right. Sorry about that. I misunderstood, 
Yes, it looks like this is missing in the .NET SDK.

Reply:
Thanks for pointing that out @joaopgrassi. This seems good to me!